Editorial suggestions for abstract and intro (#49)
marten-seemann/draft-seemann-quic-reliable-stream-reset: draft-ietf-quic-reliable-stream-reset.md

--- abstract

> QUIC transport protocol streams provide an ordered and reliable byte stream.

Reviewer: ```suggestion
QUIC transport protocol provides ordered and reliable byte streams.
```
Maybe? Just a matter of taste.

Reviewer: Or maybe: QUIC transport protocol provides byte streams that are ordered and reliable.

Author: I like the second suggestion, so tweaked it and pushed up

Once a sender has started to emit stream data, it can abort the operation using
a RESET_STREAM frame. When a sender resets a stream, it also stops
retransmitting stream data in the case of packet loss. At the receiver, there is
no guarantee that any data sent on a reset stream is delivered.

This document defines an extension that allows a sender to reset a stream with
guaranteed delivery of stream data from the beginning of the stream up to an
explicitly declared byte offset.

--- middle

# Introduction

Version 1 of the QUIC transport protocol ({{!QUIC=RFC9000}}) provides

> streams, an ordered and reliable byte stream. Once a sender has started to emit

Reviewer: @marten-seemann if you are concerned about that, maybe something like:
```suggestion
byte-oriented streams that are ordered and reliable. Once a sender has started to emit
```

It's almost a duplicate of the first sentence of the abstract, but hopefully nobody cares.

Reviewer 2: I merged too quickly. Byte-oriented? I've never heard this term before.
Maybe better: reliable byte-ordered streams.
